worksheet.rb

```diff
@@ -23,6 +42,33 @@
 # into this data structure, such as "DR0004"
 # and "3rd Feb 2016" and "RD0022"
 
+data_hash = {
```

Comment:
Instead of naming this `data_hash`, I may suggest calling this something like `drivers_data` or `drivers_info` -- That name is more descriptive about what each key/value pair is